### PR TITLE
The default value of UpdateInterval is different in Windows Server 2…

### PR DESCRIPTION
**Pull request by taka-oi**
…016.

### CODE REVIEW

**Comment by eross-msft** [comment]
@dcuomo  Please review this change and let me know if it's ready to merge for publishing. Thanks!

**Comment by eross-msft** [comment]
Sending this question to PM. Will respond after I get response.

**Comment by illfated** [comment]
@eross-msft : Do you think it would be OK to ask the PM again, just to see if we can have this confirmed before WS 2016 goes out of mainstream support?

**Comment by illfated** [comment]
@taka-oi : Would you be able to resolve the file conflict, just in case we get a positive feedback for this change?

**Comment by eross-msft** [comment]
@illfated The PM apparently isn't on GitHub. I will email him with this again and see if we can get this resolved.

**Comment by illfated** [comment]
@taka-oi : Any chance you will be returning to this Pull Request to resolve the file conflict, to make it available for merging? (I assume that there will be a reply from the MS Docs team confirming the validity of this contribution, I just don't know when.)

**Comment by illfated** [comment]
Anyway, if you find that the document changes, since you opened this PR, are causing too much trouble for you, it should be quite easy to close this PR and open a new PR containing the same change.

**Comment by PRMerger13** [comment]
@taka-oi : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by illfated** [comment]
I have tried a few times to use search engines to see if I could locate more information about the specific details regarding Windows Server 2016 and `W32Time\Config\\UpdateInterval = 100`, but I have not been successful yet. If there exists any information supporting this statement out there, it would be quite helpful to link it here in a new comment, or in the main PR comment.

File: WindowsServerDocs/networking/windows-time-service/Windows-Time-Service-Tools-and-Settings.md [hunk 1]
```diff
@@ -281,7 +281,7 @@ In the following tables, "All versions" refers to versions of Windows that inclu
...
+|**UpdateInterval** |All versions |Specifies the number of clock ticks between phase correction adjustments. The default value for domain controllers is **100**. The default value for domain members is **30,000**. The default value for stand-alone clients and servers is **360,000**. However, in Windows Server 2016, the default value is **100 in any state**.<br /><br />**Note**<br />Zero is not a valid value for the **UpdateInterval** registry entry. On computers running Windows Server 2003, Windows Server 2003 R2, Windows Server 2008, and Windows Server 2008 R2, if the value is set to **0**, the Windows Time service automatically changes it to **1**.|
```
Comment (Teresa-Motiv) [inline WindowsServerDocs/networking/windows-time-service/Windows-Time-Service-Tools-and-Settings.md:284]: Please clarify what you mean by "in any state." It should only be bold if it's part of the actual value. 
Does it mean that for Windows Server 2016, the default value is *100* whether the server is a domain controller, a domain member, or a standalone server?"

**Review by Teresa-Motiv — CHANGES_REQUESTED** [review]
Please clarify: Is the default value

**Comment by Teresa-Motiv** [comment]
@illfated were you able to confirm the default value?

**Comment by illfated** [comment]
Sorry, I have not found this value documented elsewhere, even though I wish I had found something that would clarify this issue. I may not have been diligent enough with my searches.

**Comment by Teresa-Motiv** [comment]
Reassigning this PR--the article is currently being rewritten by another team, and that may resolve this issue.

**Comment by illfated** [comment]
#please-close

**Pull request closed without merging**